CoinDock-api/app/Http/Resources/V1/coinCardResource.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Resources\V1;
+
+use App\Models\V1\Coin;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+
+class coinCardResource extends JsonResource
```

Comment:
It's CoinCardResource not coinCardResource.

Reply:
Changed